core-bundle/src/Security/TwoFactor/Authenticator.php

```diff
@@ -55,7 +55,7 @@ public function getProvisionUri(User $user, Request $request): string
     public function getQrCode(User $user, Request $request): string
     {
         $renderer = new ImageRenderer(
-            new RendererStyle(180, 0),
+            new RendererStyle(180, 3),
```

Comment:
```suggestion
            new RendererStyle(180, 1),
```
I'd suggest using 1 for now as it's the closest to how it looked like before.